plugins/fixtures/routes.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  storeFixtures: {
+    path: '/fixtures/data',
+    method: 'GET',
```

Comment:
The method should be `POST`. `GET` request are supposed to not change anything per https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html